exporter/metric/metric.go

```diff
@@ -579,19 +579,28 @@ func sumToTimeSeries[N int64 | float64](point metricdata.DataPoint[N], metrics m
 	}, nil
 }
 
-func histogramToTimeSeries[N int64 | float64](point metricdata.HistogramDataPoint[N], metrics metricdata.Metrics, mr *monitoredrespb.MonitoredResource) (*monitoringpb.TimeSeries, error) {
+//nolint:revive
```

Comment:
is this for the control-coupling lint check? Could you add a comment/todo for that? I did the same in https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/a3fa73dadda89df41b0641e74e796145f134a347/exporter/collector/monitoredresource.go#L59-L61

Reply:
done